gui/control.js

```diff
@@ -17,13 +17,44 @@ const addAccountButton = document.getElementById('add-account-btn');
 window.addEventListener('unload', beforeQuit)
 window.addEventListener('DOMContentLoaded', main)
 
+const loadingMessage = document.getElementById('loading-message');
+const loadingOverlay = document.getElementById('loading-overlay');
+const showOverlay =false;
 
-async function addButtonPressed() {
+function setLoadingOverlayVisibility(showOverlay) {
+  loadingOverlay.style.display = showOverlay ? 'flex' : 'none';
+}
```

Comment:
if this is no longer needed, pls remove it

Reply:
Removed